app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/activity_details.scala.html

```diff
@@ -82,7 +82,7 @@
         }
         @if(request.hasPermission(Permission.MOVE_CASE_BACK_TO_QUEUE) && reassignCaseStatuses.contains(c.status)){
             <span class="column-one-third mt-1">
-                <a tabindex="@{startAtTabIndex+13}" id="reassign-queue-link" href="@ReassignCaseController.showAvailableQueues(c.reference, request.uri)">
+                <a tabindex="@{startAtTabIndex+213}" id="reassign-queue-link" href="@ReassignCaseController.showAvailableQueues(c.reference, request.uri)">
```

Comment:
213?  Are you sure?

Reply:
Gives us a possible 200 to use for any events (so 100 events), as this appears afterwards.

Reply:
I made it a nicer round number anyway